docs/layers.md

```diff
@@ -806,29 +807,35 @@ To resolve inconsistency in data tagging in OpenStreetMap we normalize several o
 * `block`
 * `boat_rental`
 * `boat_storage`
-* `bollard`
 * `boatyard`
 * `boat_lift`
+* `bollard`
```

Comment:
I don't see a test or pois.yaml change for `bollard`.

Reply:
Yup, sorry, it's not clear, but I only moved `bollard` from [between `boat_storage` and `boatyard`](https://github.com/tilezen/vector-datasource/pull/1468/files/c01d34b1f76e665c902d8b5561febe6722aca795#diff-606a00db54cec09bedc8a0a9e6e26a15L808) to between `boat_lift` and `bookmaker` to keep it in proper alphabetical order.